### ISSUE TITLE
Indonesia GTM - Team 4: Localization

### ISSUE THREAD

**Issue by RizRiyz** [mozilla/webmaker-launch#47]
Goal:
- 100 percent localizations of Webmaker and Mozilla Learning site

Team Member: @fauzanalfi, @kelimuttu, @technowizard
Team Pilot: @kelimuttu

Plan: https://docs.google.com/document/d/1Gq0ZRjMKo7xdyUAdyd5kndFRxv3ULrHkDPW3PujyEdw/edit?usp=sharing

**Comment by fauzanalfi** [comment]
@secretrobotron:
Here, I want to explain more why we would like to do remote and on-site L10n hackathon.
First, we want to make on-site L10n hackathon is 'a celebration' to people's participation to Webmaker launch project. When all participants are reaching the goal of 100% localized material, they will feel achieving something in this project. We want them to get those feeling, not just to have everything localized.

First, to make the on-site hackathon much easier and faster, we want to have at least 25% of materials are localized by having remote hackathon with registered Mozillians and Reps. So, we just need to localize at least 75% more in on-site hackathon which is much easier to accomplish.

Hopefully you get the idea :)

**Comment by kelimuttu** [comment]
@secretrobotron Bobby, can you the share us the link of the translation material from transifex here?

**Comment by kelimuttu** [comment]
I think I've found it. Is this the right link? @secretrobotron => https://www.transifex.com/projects/p/webmaker/translate/#id/webmaker-stringsxml/50935942

**Comment by secretrobotron** [comment]
@kelimuttu that's right! And https://www.transifex.com/projects/p/webmaker/resource/user_createdjson/

I'll post those on the wiki right now.

**Comment by secretrobotron** [comment]
Thanks for the wait, @kelimuttu. I'll continue to put relevant localization info here: https://github.com/mozilla/webmaker-launch/wiki/Localization

**Comment by fauzanalfi** [comment]
I've written some updates here: https://github.com/mozilla/webmaker-launch/wiki/Indonesia-Updates

TL;DR: webmaker-mailroom and webmaker-android are already localized. Now, we're waiting for 'webmaker landing page' and 'Google Play Store' material to be localized

**Comment by secretrobotron** [comment]
@fauzanalfi @kelimuttu 

We've got a couple more assets that we're hoping to get localized this week so we can print them. Do you think you can help?

We're making a poster, and there are a couple parts of it that need to be translated into sensible Indonesian: https://github.com/mozilla/webmaker-launch/issues/53#issuecomment-117377754
- "Sunset with Friends"
- "Discover. Create. Share."
- "Webmaker lets you build original creations on your mobile device in minutes and share them with your friends. Download Webmaker for Android — it's free and open source."

cc @kevzawacki @edrushka

**Comment by fauzanalfi** [comment]
@secretrobotron 
Sure, we can help you to localize them. We will post localized parts ASAP.

**Comment by fauzanalfi** [comment]
@secretrobotron 
Since we're very excited with Webmaker app, we're finally localize those parts for the poster. Here they are:
- "Senja bersama Teman"
- "Temukan. Buat. Bagikan"
- "Webmaker dapat kamu gunakan untuk membuat kreasi orisinil di perangkat bergerakmu dalam sekejap dan membagikannya ke teman-temanmu. Unduh Webmaker untuk Android – semuanya gratis dan kode sumbernya terbuka"

Please let us know if there are another parts that we must localize to Indonesian ;)

**Comment by secretrobotron** [comment]
Thanks @fauzanalfi. Super helpful.

**Comment by secretrobotron** [comment]
Just noticed that the translations for [https://www.transifex.com/projects/p/webmaker/resource/webmaker-stringsxml/](webmaker-strings) aren't reviewed. Are you able to review them (maybe go through the [QA checklist](https://gist.github.com/k88hudson/16e0b6a720f33e1f7c3e)) to bump that up?

**Comment by KevZawacki** [comment]
Hi @fauzanalfi, hoping you can assist with one more translation. 
We'd like to translate "text size," in the context of this post (first screen): https://goo.gl/Z1R2Il
Thanks!

cc @lovegushwa

**Comment by kelimuttu** [comment]
@secretrobotron I'm working on it. I'll let you know as soon as it finished. 

@KevZawacki "text size" is "Ukuran Teks" in Bahasa Indonesia. Any other text we need to translate?
cc @lovegushwa

**Comment by lovegushwa** [comment]
@fauzanalfi @KevZawacki @secretrobotron @cassiemc 
Please review the final placed copy. Thank you!

![webmaker-poster-27x39in-g3](https://cloud.githubusercontent.com/assets/12228435/8493317/5818ac8c-210d-11e5-9647-3e790ce9395a.jpg)

**Comment by fauzanalfi** [comment]
@lovegushwa:
That's perfect!

By the way, we want to fix the bottom part of the poster and make it into more effective sentence.
So...
- "Webmaker dapat kamu gunakan untuk membuat kreasi orisinil di perangkat bergerakmu dalam sekejap dan membagikannya ke teman-temanmu. Unduh Webmaker untuk Android – semuanya gratis dan kode sumbernya terbuka"

will be change to:
- "Webmaker dapat kamu gunakan untuk membuat kreasi orisinil di perangkat bergerak dalam sekejap dan membagikannya ke teman-teman. Unduh Webmaker untuk Android – semuanya gratis dan kode sumbernya terbuka"

Thanks.

/cc @kelimuttu @KevZawacki @secretrobotron @cassiemc

**Comment by lovegushwa** [comment]
Thank you so much for the translation and help @fauzanalfi !!!
Please review the poster and postcards for accuracy. Files will release to the printer tomorrow morning the 7th (Indonesian time.) 

POSTER
![webmaker-poster3](https://cloud.githubusercontent.com/assets/12228435/8532780/9071bb20-23e5-11e5-965a-a1d02df5dd07.jpg)

POSTCARD FRONT
![webmaker-postcard-5x7in-proof5](https://cloud.githubusercontent.com/assets/12228435/8532790/9a8fe032-23e5-11e5-8087-a79ddd06b54f.jpg)

POSTCARD BACK
![webmaker-postcard-5x7in-proof6](https://cloud.githubusercontent.com/assets/12228435/8532794/a25462ca-23e5-11e5-869f-a7520dc590de.jpg)

CC @LauraReynal @secretrobotron @KevZawacki

**Comment by fauzanalfi** [comment]
@lovegushwa both poster and postcard are great!!!

But, @secretrobotron and I found out that there's a missing period after "Bagikan" both on poster and postcard. Can you fix it please?

Thanks.

**Comment by lovegushwa** [comment]
@fauzanalfi 
The original copy translated to Indonesian is missing a period at the end as well. Also any text with multiple sentences is missing a period on the last sentence. I though perhaps this was Indonesian grammar? Please advise!

 "Senja bersama Teman"
 "Temukan. Buat. Bagikan"
 "Webmaker dapat kamu gunakan untuk membuat kreasi orisinil di perangkat bergerakmu dalam sekejap dan membagikannya ke teman-temanmu. Unduh Webmaker untuk Android – semuanya gratis dan kode sumbernya terbuka"

CC @secretrobotron @KevZawacki @LauraReynal

**Comment by fauzanalfi** [comment]
@lovegushwa @secretrobotron
oops sorry, it's my bad. There should be a period after 'Bagikan'. Btw, there're last minute changes on the translation. Is it OK if I'll inform it to you maybe in an hour? I'm still gathering feedback from whole Indonesia launch team members.

**Comment by lovegushwa** [comment]
Thanks @fauzanalfi ! 
Should all sentences in Indonesian end in a period?
No problem with the translation changes, I will check back soon.
cc @secretrobotron

**Comment by fauzanalfi** [comment]
@lovegushwa
Yes, mostly all sentences in Indonesian end in a period.
Cc @secretrobotron

**Comment by fauzanalfi** [comment]
@lovegushwa
Here's updated version of last sentence on poster & postcard:
"Webmaker dapat kamu gunakan untuk membuat kreasi orisinal di perangkat bergerak dalam sekejap dan membaginya pada teman-teman. Unduh Webmaker untuk Android — gratis dan sumbernya terbuka."

cc @secretrobotron

**Comment by lovegushwa** [comment]
Thank you, Thank you @fauzanalfi 
cc @secretrobotron

**Comment by lovegushwa** [comment]
@fauzanalfi 
Please confirm the  revised version of the poster and postcard.

POSTER
![webmaker-poster3](https://cloud.githubusercontent.com/assets/12228435/8535280/3b819c98-23f9-11e5-9d38-0c61937fe53d.jpg)

POSTCARD FRONT
![webmaker-postcard-5x7in-revised](https://cloud.githubusercontent.com/assets/12228435/8535311/7fa19ad6-23f9-11e5-96f4-84aa8d6ece26.jpg)

POSTCARD BACK
![webmaker-postcard-5x7in-revised2](https://cloud.githubusercontent.com/assets/12228435/8535316/88844770-23f9-11e5-81ff-fd04b07e92d8.jpg)

**Comment by fauzanalfi** [comment]
@lovegushwa
Awesome! But, there's a missing period after 'Bagikan' on poster. It should be like 'Bagikan' on postcard which has period at the end.

Overall, it's super awesome!

Thanks a lot :D
/cc @secretrobotron

**Comment by lovegushwa** [comment]
lol @fauzanalfi I only fixed one, thanks!

**Comment by kelimuttu** [comment]
Dear @secretrobotron, there's a mistake on the translation in Google play store. It written "Buat. Temukan. Share. Sebuah menyenangkan, cara mudah untuk berpartisipasi di Web."
![photo_2015-07-13_13-06-55](https://cloud.githubusercontent.com/assets/4922388/8643640/c2d7e956-2960-11e5-974c-3af6e3c4fad2.jpg)
It ruin the meaning, lol. It should be 'Buat. Temukan. Bagikan. Sebuah cara mudah dan menyenangkan untuk berpartisipasi di Web.'

cc: @fauzanalfi

**Comment by secretrobotron** [comment]
Thanks @kelimuttu!

@HPaulJohnson @thisandagain how did these get translated? Through play store system? How can we fix?

**Comment by KevZawacki** [comment]
Hi @kelimuttu and @fauzanalfi,

We're finalizing a few more Webmaker assets (https://github.com/mozilla/webmaker-launch/issues/34), and hoping you can help translate a few phrases:

-Download The App
-Download
-Webmaker lets you build and share original content on your smartphone. Start creating in minutes.
-Learn More

Thanks for your help!
cc @jaykappa

**Comment by fauzanalfi** [comment]
Hi @KevZawacki,

Here are the translated phrases (in the same order):
- Unduh Aplikasinya
- Unduh
- Webmaker membantumu membuat dan membagikan konten orisinal di ponsel pintarmu. Mulailah berkreasi dalam hitungan menit.
- Pelajari Lebih Lanjut

/cc @kelimuttu @secretrobotron

**Comment by KevZawacki** [comment]
Thanks so much, @fauzanalfi!

cc @jaykappa

**Comment by jaykappa** [comment]
@fauzanalfi would you mind reviewing the following to ensure translation was applied appropriately:

![2015-webmaker-twitter-1760x800-ind](https://cloud.githubusercontent.com/assets/1036635/8707673/3f018118-2b05-11e5-8da1-c6de5e4254e0.jpg)
![2015-webmaker-webbanners-horz-468x60-ind](https://cloud.githubusercontent.com/assets/1036635/8707672/3f018406-2b05-11e5-920e-9b5748acac7f.jpg)
![2015-webmaker-webbanners-horz-728x90-v3-ind](https://cloud.githubusercontent.com/assets/1036635/8707671/3f00069e-2b05-11e5-9f3a-00dbbc193e38.jpg)

cc @KevZawacki @secretrobotron @lovegushwa

**Comment by fauzanalfi** [comment]
@jaykappa sure. I and Indonesian team have reviewed those following assets and everything is perfect. Thanks a lot, Jay!

cc @kelimuttu @KevZawacki @secretrobotron @lovegushwa

**Comment by jaykappa** [comment]
thank you @fauzanalfi + team

**Comment by secretrobotron** [comment]
@kelimuttu @fauzanalfi uploaded google play strings to transifex: https://www.transifex.com/projects/p/webmaker/resource/google-play/

**Comment by kelimuttu** [comment]
@secretrobotron cool! I'll keep you updated once we finish it.

**Comment by secretrobotron** [comment]
@kelimuttu btw, we _just_ uploaded another transifex file to handle the desktop version of the app: https://www.transifex.com/projects/p/webmaker/resource/betawebmakerorg/

**Comment by kelimuttu** [comment]
@secretrobotron okay. Working on it (;

**Comment by secretrobotron** [comment]
@kelimuttu @fauzanalfi our localization specialist, @alicoding was able to finish the client-side code today. So, we can QA the localized app now! :)

The team had to switch over to a different transifex resource, and the dev team has added some more strings as the app has progressed over the past couple weeks: https://www.transifex.com/projects/p/webmaker/resource/webmaker-core/

Whenever you get the opportunity, that can be fully translated now too, and we can see the changes in the app ASAP :boom: :boom: :boom: !

Sound alright?

**Comment by fauzanalfi** [comment]
@secretrobotron that's OK. We will notify you ASAP if we're fully translated them.

/cc @kelimuttu @alicoding

**Comment by fauzanalfi** [comment]
@secretrobotron I want to inform that google-play, betawebmakerorg and webmaker-core are fully translated.

cc @kelimuttu

**Comment by alamanda** [comment]
dear @kelimuttu @fauzanalfi would it be possible for Team04 to localize this teaching kit: http://mozilla.github.io/webmaker-curriculum/MobileWeb/create-webmaker-project.html for Team02 and Team06? They will use it for presentation and hands-on workshop during school visit.
thank you!

**Comment by fauzanalfi** [comment]
@alamanda we will try, Ina :)
/cc @kelimuttu @technowizard

**Comment by secretrobotron** [comment]
Cool! Let me know if I can clarify anything in that teaching kit.

**Comment by alamanda** [comment]
@kelimuttu @fauzanalfi any news for this? the first school visit will be in August 1st. we still have 3 days until then:) please let me know if I could help also to review it once you have translate the document. 
thank you so much!

**Comment by KevZawacki** [comment]
Hi @fauzanalfi,

We're putting Indonesia copy in our promotional Webmaker video, and are hoping you can help with translation. Specifically, the phrase: "my first blog." Thanks!

cc @lovegushwa @HPaulJohnson

**Comment by kelimuttu** [comment]
@KevZawacki  "my first blog." in Bahasa Indonesia is "blog pertamaku." 

cc @fauzanalfi  @lovegushwa @HPaulJohnson

**Comment by KevZawacki** [comment]
Thank you, @kelimuttu!

**Comment by KevZawacki** [comment]
Hi all, we're preparing Webmaker copy for Firefox snippets, and are hoping you can help to localize:

Unleash your creativity with Mozilla's free, open source Webmaker app for Android. Get the scoop on how to become a digital creator with Webmaker

Discover. Create. Share. Make personal scrapbooks, photo galleries, comic strips and more with the free Webmaker app. Download it today

The all-new Webmaker app from Mozilla: the simplest way to tell your story on mobile. Download it for free today and learn the web in minutes

A new way to participate on the mobile Web. The all-new Webmaker from Mozilla: an app to create a Web for you, by you. Download it for free today

An app made for storytellers, learners and community. The all-new Webmaker app from Mozilla: a better way to participate on the mobile Web. Download it for free today

Discover original content from the world around you. Create, remix and reimagine how you tell your story on the Web. Share the things you love with your friends, family and community. The all-new Webmaker app from Mozilla. Download it for free today 

Thank you! cc @HPaulJohnson

**Comment by KevZawacki** [comment]
Thanks for the quick translation work in Telegram! Ported to below:

## 

Pancarkan kreativitasmu dengan Webmaker app untuk Android gratis dan sumber kode terbuka dari Mozilla. Cari tahu caranya untuk menjadi seorang pencipta digital dengan Webmaker

Temukan. Buat. Bagikan. Buat buku tempel pribadi, galeri foto, komik strip dan lainnya dengan Webmaker app yang gratis. Unduh sekarang juga

Aplikasi Webmaker terbaru dari Mozilla: cara termudah untuk menceritakan ceritamu lewat mobile. Unduh gratis sekarang juga dan pelajari web dalam hitungan menit

Cara baru untuk berpartisipasi pada mobile Web. Webmaker terbaru dari Mozilla: sebuah aplikasi untuk membuat sebuah Web untuk kamu, dari kamu. Unduh gratis sekarang juga.

Sebuah aplikasi untuk para penutur cerita, pembelajar dan komunitas. Aplikasi Webmaker terbaru dari Mozilla: cara yang lebih baik untuk berpartisipasi dalam mobile Web. Unduh gratis sekarang juga

Temukan konten orisinal dari dunia di sekitarmu. Buat, olah-kembali, dan bayangkan kembali bagaimana kamu menceritakan ceritamu di Web. Bagikan hal-hal yang kamu sukai bersama teman-teman, keluarga, dan komunitas. Aplikasi Webmaker terbaru dari Mozilla. Unduh gratis sekarang juga

**Comment by kelimuttu** [comment]
@KevZawacki We actually have made a few correction for the first paragraph in another telegram group. Here it is : 

Pancarkan kreativitasmu dengan Webmaker app untuk Android dari Mozilla yang gratis dan kode sumbernya terbuka. Cari tahu caranya untuk menjadi seorang pencipta digital dengan Webmaker

cc// @fauzanalfi

**Comment by KevZawacki** [comment]
Thank you, @kelimuttu. I'll let @HPaulJohnson know.

**Comment by alamanda** [comment]
@kelimuttu @fauzanalfi would you please help with this? 
https://etherpad.mozilla.org/WebmakerBlog
I think it will be best if we can run it this week. thanks!:D

**Comment by alamanda** [comment]
My bad! I just realized that it has been translated and posted in the Mozilla Indonesia website:)) OMG! I'm sorry for this!

**Comment by kelimuttu** [comment]
@alamanda I'm just about to reply it btw. That's okay. (;

**Comment by KevZawacki** [comment]
Hi all, our video team put together a neat video about Webmaker: https://www.youtube.com/watch?v=46dYHphb2Q8

If you think this would be useful in your region, our video team can re-edit a localized version if you're able to share copy translations. Let me know!

**Comment by alamanda** [comment]
Hi @KevZawacki I tried to watch the video but it says: the video has been removed by the user. 
![screen shot 2015-08-26 at 10 32 50 am](https://cloud.githubusercontent.com/assets/12945788/9485021/fb0468aa-4bdd-11e5-92b7-da99f598938b.png)

**Comment by KevZawacki** [comment]
Ah, sorry about that, @alamanda! I'll track down a proper link. :blush:

**Comment by KevZawacki** [comment]
Hey @alamanda, here's the latest video: https://www.youtube.com/watch?v=hwprnE6ySsE&feature=youtu.be
